site/docs-md/guides/ionic-framework-app.md

```diff
@@ -171,7 +171,7 @@ Some Capacitor plugins, including the Camera, provide the web-based functionalit
 npm install @ionic/pwa-elements
 ```
 
-**Note:** There's currently an issue with importing the Ionic PWA elements library in an Ionic Angular app. In the meantime, include this script tag in `src/index.html`: 
+Include this script tags in `src/index.html`:
```

Comment:
Could we actually remove this? I'd prefer just to show the Import way. They can always find the scripts on pwa elements docs page.

Reply:
removed